src/shared/schema/user.schema.ts

```diff
@@ -22,50 +26,59 @@ import {
   redisSet,
   sendMail,
 } from '../utils';
-import { BadRequestException } from '@nestjs/common';
-import { CreateUserDto } from '../dtos/create-user.dto';
-import { UserInviteDto } from 'src/users/dto/user-invite.dto';
 
 export type UserDocument = HydratedDocument<User>;
 
+// example structure (request from frontend)
+// {
+//   _id: ObjectId,
+//   'email': string,
+//   'pssword': string,
+//   'role': string[],
+//   'basic_info':{
+//     'firstName': string,
+//     'lastName': string,
+//     'profileSummary': string,
```

Comment:
@snebo : Recommendation: It would be great to remove unused code so we don't create a future code smell-(Technical Debt)